contracts/SaveDAI.sol

```diff
@@ -60,51 +59,23 @@ contract SaveDAI is ISaveDAI, ERC20, ERC20Detailed, Pausable {
         uniswapFactory = UniswapFactoryInterface(uniswapFactoryAddress);
         daiUniswapExchange = _getExchange(daiAddress);
         ocDaiExchange = _getExchange(ocDaiAddress);
+        _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
```

Comment:
@dougiebuckets might be better to be explicit here and use the Pauser role provided in OpenZeppelin?